develop-docs/self-hosted/experimental/reverse-proxy.mdx

```diff
@@ -71,6 +71,21 @@ server {
     proxy_buffer_size    128k;
     proxy_buffers        4 256k;
 
+    # Handle CORS-related headers for ingest endpoints.
```

Comment:
Why is this necessary? It would be helpful to understand why these changes are being made to see if this is indicative of a larger issue, especially since it's not something common we've seen

Reply:
Linking the discord thread could give helpful context

Reply:
AFAIK it was this: https://discord.com/channels/621778831602221064/1334066733752909855

Oh, also here, by myself: https://discord.com/channels/621778831602221064/796028405833007104/1169197966460985395

![image](https://github.com/user-attachments/assets/0da3275a-d9b4-4106-bf14-7ad987d6635b)